jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -151,8 +151,27 @@ public interface Request extends Attributes, Content.Source
     Context getContext();
 
     /**
-     * TODO see discussion in #7713, as this path should probably be canonically encoded - ie everything but %25 and %2F decoded
-     * @return The part of the decoded path of the URI after any context path prefix has been removed.
+     * Get the full context path.
+     * @return The full contextPath of the request. In the common case, this is equivalent to
```

Comment:
maybe reword to get rid of "common" here as well:
```suggestion
     * @return The full contextPath of the request. Unless using nested contexts, this is equivalent to
```